packages/linux-drivers/mali-bifrost/package.mk

```diff
@@ -0,0 +1,33 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="mali-bifrost"
+PKG_ARCH="arm aarch64"
+PKG_LICENSE="GPL"
```

Comment:
Small nit - could you move `PKG_ARCH` and `PKG_LICENSE` between `PKG_SHA256` and `PKG_SITE` - for consistency! :)